app/admin_views.py

```diff
@@ -0,0 +1,88 @@
+import os
+
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.authtoken.models import Token
+from rest_framework.permissions import AllowAny, IsAuthenticated
+
+from django.contrib.auth import authenticate, login, logout
+
+from .models import User, Question, Answer
+from .serializers import QuestionSerializer
+
+# View for Admin Signup
+class AdminSignupView(APIView):
+    permission_classes = (AllowAny,)
+
+    def post(self, request):
+        req_data = {}
```

Comment:
Why is this req_data being used. `request.data` can be used without reassignment